lib/terraorg/model/org.rb

```diff
@@ -202,13 +202,14 @@ def get_squads_md
     md_lines.join("\n")
   end
 
-  def generate_tf
-    tf = @member_platoons.map { |p| p.generate_tf(@id) }.join("\n")
-    File.write('auto.platoons.tf', tf)
+  # def generate_tf_platoons
+  #   @member_platoons.map { |p| p.generate_tf(@id) }.join("\n")
 
-    tf = @member_exception_squads.map { |s| s.generate_tf(@id) }.join("\n")
-    File.write('auto.exception_squads.tf', tf)
+  # def generate_tf_sqauds
```

Comment:
Why commented out?

Reply:
No idea why I did that.  Fixed in current commit